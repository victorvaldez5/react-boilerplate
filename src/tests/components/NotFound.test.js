import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFound';

test("Should test the not found page matches not found page snapshot", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});