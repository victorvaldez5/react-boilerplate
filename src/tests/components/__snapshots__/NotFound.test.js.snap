// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test the not found page matches not found page snapshot 1`] = `
<div>
  <h1>
    404! Page Not Found!
  </h1>
  <h3>
    Return to 
    <Link
      replace={false}
      to="/"
    >
      Dashboard
    </Link>
  </h3>
</div>
`;
